validator/src/commands/run/args/rpc_bigtable_config.rs

```diff
@@ -1,10 +1,16 @@
 use {
     crate::commands::{FromClapArgMatches, Result},
-    clap::{value_t, ArgMatches},
+    clap::{value_t, Arg, ArgMatches},
+    solana_clap_utils::{hidden_unless_forced, input_validators::is_parsable},
     solana_rpc::rpc::RpcBigtableConfig,
     std::time::Duration,
 };
 
+pub(crate) const DEFAULT_BIGTABLE_INSTANCE_NAME: &str = "solana-ledger"; // solana_storage_bigtable::DEFAULT_INSTANCE_NAME.to_string(),
+pub(crate) const DEFAULT_BIGTABLE_APP_PROFILE_ID: &str = "default"; // solana_storage_bigtable::DEFAULT_APP_PROFILE_ID
+pub(crate) const DEFAULT_BIGTABLE_TIMEOUT: &str = "30";
+pub(crate) const DEFAULT_BIGTABLE_MAX_MESSAGE_SIZE: &str = "67108864"; // (64 * 1024 * 1024), rpc_bigtable_max_message_size: solana_storage_bigtable::DEFAULT_MAX_MESSAGE_SIZE:
```

Comment:
A couple items;
1. Doesn't need to be `pub(crate)`
2. This is a very long line (176 characters); fmt/clippy should have complained about this in CI. Can you dig into why it did not
3. I don't like duplicating the constant here. If the constant changes, this value is likely to be forgotten. I think we should do the same approach we did for `PubSub` args. An alternative could also be adding a unit test to enforce behavior, but I'd say let's do the lazy lock approach for consistency.

Reply:
yeah, I just pushed 4 new commits that update these 4 default values.

1. check pub(crate)
2. use the reference value if it exists
3. add unit tests to ensure the values unchanged

Reply:
for the lint issue, I guess they always have some bugs haha. will take a look into it if I don't have other higher priority 🫡